src/redux/user/userAPI.js

```diff
@@ -0,0 +1,49 @@
+import { addUSER, removeUSER } from './user';
+
+const SIGNUP = 'https://booking-tour-app.herokuapp.com/signup';
+const LOGIN = 'https://booking-tour-app.herokuapp.com/login';
+
+export const login = (data) => async (dispatch) => {
+  const response = await fetch(LOGIN, {
+    method: 'POST',
+    headers: { 'Content-Type': 'application/json' },
+    body: JSON.stringify(data),
+  });
+  console.log(response.status);
```

Comment:
- [ ]  Please, kindly remove this old piece of debugging code from the repo (console.log).  That way, it will be cleaner.  💪